.github/workflows/webservices.yml

```diff
@@ -0,0 +1,12 @@
+on: repository_dispatch
+
+jobs:
+  webservices:
+    runs-on: ubuntu-latest
+    name: webservices
+    steps:
+      - name: webservices
+        id: webservices
+        uses: conda-forge/webservices-dispatch-action@master
+        with:
```

Comment:
Could you please delete those workflows?
Because the feedstock will be generated automatically and it will not pull those things here
You will just need to invoke the bot on the feedstock
https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-bot-automerge

Reply:
I see, ok, done.